src/js/d3_svg_cell_renderer.js

```diff
@@ -0,0 +1,207 @@
+var utils = require('./utils');
```

Comment:
I see that point. I also kind of aesthetically like seeing all lowercase filenames though. @jjgao what do you think?